### PR TITLE
feature/1255/show-file-count-in-side-bar

### PR DESCRIPTION
**Pull request by shaman-apprentice**
# Show file count of folders in Attribute-Side-Bar

_**Please read the [CONTRIBUTING.md](https://github.com/MaibornWolff/codecharta/blob/main/CONTRIBUTING.md) before opening a PR.**_

Closes: #1255 

## Description
s.  #1255 and screenshot below

## Screenshots or gifs
image

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -9,6 +9,8 @@ and this project adheres to [Semantic Versioning](http://semver.org/)
 
 ### Added 🚀
 
+-   Show file count of folders in Attribute-Side-Bar
```
Comment (RomanenkoVladimir) [inline CHANGELOG.md:12]: Please include the issue number next to the change, also for added features we decided to try for a bit more elaborate texts to help users catch up.

File: visualization/app/codeCharta/ui/attributeSideBar/attributeSideBar.component.spec.ts [hunk 2]
```diff
@@ -119,6 +120,13 @@ describe("AttributeSideBarController", () => {
 
 			expect(attributeSideBarController["_viewModel"].fileName).toEqual("my_fileName")
 		})
+
+		it("should has fileCount in it's _viewModel", () => {
```
Comment (RomanenkoVladimir) [inline visualization/app/codeCharta/ui/attributeSideBar/attributeSideBar.component.spec.ts:124]: nit: should have

File: CHANGELOG.md (hunk 1 above)
Comment (shaman-apprentice) [reply CHANGELOG.md:12]: What do you think about adding the quick screenshot from PR description?

**Review by RomanenkoVladimir — APPROVED** [review]
LGTM, just included a few small nits.
Also we might wanna add a

File: visualization/app/codeCharta/ui/attributeSideBar/attributeSideBar.component.ts [hunk 3]
```diff
@@ -40,12 +40,14 @@ export class AttributeSideBarController
...
 		isSideBarVisible: boolean
+		fileCount: number
 	} = {
 		node: null,
 		fileName: null,
 		primaryMetricKeys: { node: {}, edge: {} } as PrimaryMetrics,
 		secondaryMetricKeys: null,
-		isSideBarVisible: null
+		isSideBarVisible: null,
+		fileCount: 0
```
Comment (ce-bo) [inline visualization/app/codeCharta/ui/attributeSideBar/attributeSideBar.component.ts:50]: We could improve the name a little bit. Suggestion: `packageFileCount`. Feel free to update the name.

**Review by ce-bo — APPROVED** [review]

**Comment by shaman-apprentice** [comment]
> Also we might wanna add a test to show it wont display for folders?

Would love to use something like [this html testing](https://docs.angularjs.org/guide/unit-testing). But for now out of scope. I will definitely play around with it sometime :) - If you can show me a quick way with current setup to test it, it would be very appreciated.

File: CHANGELOG.md (hunk 1 above)
Comment (shaman-apprentice) [reply CHANGELOG.md]: And it is added :)

Comment (RomanenkoVladimir) [reply CHANGELOG.md]: Good idea, we might use that from now on

**Comment by RomanenkoVladimir** [comment]
Two more nits: 
1. we use Titles like Feature/number/descirption for PRs
2. in the PR text it say Issue: number, please change that to closes: or another closing keyword so git updates Zenhub when it gets merged :)

**Pull request merged**